giles-eco/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
same here, please don't change the indentation. how would I know what you've actually changed? and 4 spaces is how much it should be indented.